go-update/src/test/java/org/reactome/release/goupdate/DuplicateReporterIT.java

```diff
@@ -67,18 +72,20 @@ public void testGetDuplicateReferrers() throws Exception
 		for (String accession : duplicates.keySet())
 		{
 			assertNotNull(duplicates.get(accession));
+			totalDuplicatesCount = duplicates.keySet().size();
```

Comment:
Why does totalDuplicatesCount get reassigned in each loop iteration?  When the value is printed at the end, it will only have the value from the last cycle of the loop.